cmd/manager/main.go

```diff
@@ -59,9 +60,12 @@ func main() {
 	var agentInstallAll bool
 	var featureGatesFlags map[string]bool
 
+	logger := klogr.New()
+	klog.SetOutput(os.Stdout)
+	klog.InitFlags(flag.CommandLine)
 	flag.StringVar(&metricsAddr, "metrics-bind-address", ":38080", "The address the metric endpoint binds to.")
 	flag.StringVar(&probeAddr, "health-probe-bind-address", ":38081", "The address the probe endpoint binds to.")
-	flag.StringVar(&addonAgentImageName, "agent-image-name", "yue9944882/managed-serviceaccount-addon-agent:v0.0.0",
+	flag.StringVar(&addonAgentImageName, "agent-image-name", "quay.io/open-cluster-management/managed-serviceaccount:latest",
```

Comment:
Are we for sure that this doesn't make managed clusters actually try to pull this open-source image?

Reply:
aren't we replacing the `agent-image-name` in chart?

Reply:
I saw this in deployment
```
          args:
            - --leader-elect=true
            - --agent-image-name={{ .Values.image }}:{{ .Values.tag }}
```

Reply:
Yeah, ironically I just worked on this part. 

Thanks! I saw a hard-coded image name and got a bit freaked out.